projects/eslint-plugin/rules/strict-doc-example-extensions.js

```diff
@@ -0,0 +1,71 @@
+const getTypeName = require('../utils/get-type-name');
+
+const MESSAGE_ID = `strict-doc-example-extensions`;
+const ERROR_MESSAGE = `This mapping has wrong extension`;
+
+const DOC_EXAMPLE_INTERFACE_NAME = 'TuiDocExample';
+
+/**
+ * @type {Record<string, string>}
+ */
+const fileNameToExtension = {
+    TypeScript: '.ts',
+    HTML: '.html',
+    LESS: '.less',
```

Comment:
what about?

`**/*.{ts,less,scss,js,html}`: '.{ts,less,scss,js,html}',

```ts
'dialog-example/dialog-example.style.less': import(
            './examples/2/dialog-example/dialog-example.style.less?raw'
        ),
```

Reply:
I've supported `path2path` mappings. however, I assume that the cases on the screenhot are invalid since the extension should be provided

![image](https://user-images.githubusercontent.com/22116465/215290839-a3036316-86e8-48a2-a4f1-760c6b040632.png)